deployment/view/offramp.go

```diff
@@ -3,17 +3,37 @@ package view
 import (
 	"context"
 	"fmt"
+	"runtime"
+	"sync"
 
 	cldf_ton "github.com/smartcontractkit/chainlink-deployments-framework/chain/ton"
+	"github.com/smartcontractkit/chainlink-ton/pkg/ccip/bindings/offramp"
 	"github.com/xssnick/tonutils-go/address"
 	"github.com/xssnick/tonutils-go/ton"
+	"golang.org/x/sync/errgroup"
 
 	"github.com/smartcontractkit/chainlink-ton/pkg/ccip/bindings/common"
 )
 
 type OffRampView struct {
 	MetaData
-	// TODO add remaining fields once offramp getters are implemented in the contract
+	LatestPriceSequenceNumber uint64                              `json:"latestPriceSequenceNumber,omitempty"`
+	Config                    OffRampConfig                       `json:"Config,omitempty"`
+	SourceChainConfigs        map[uint64]OffRampSourceChainConfig `json:"sourceChainConfigs,omitempty"`
+}
+
+type OffRampSourceChainConfig struct {
+	Router                    string `json:"router,omitempty"`
+	IsEnabled                 bool   `json:"isEnabled,omitempty"`
+	MinSeqNr                  uint64 `json:"minSeqNr,omitempty"`
+	IsRMNVerificationDisabled bool   `json:"isRMNVerificationDisabled,omitempty"`
+	OnRamp                    string `json:"onRamp,omitempty"`
+}
+
+type OffRampConfig struct {
```

Comment:
I'd suggest to try to package these view data types in a similar/same structure as contract bindings where we have a named component `offramp.Config`. 

To match we could restructure code here as `offrampv.Config` vs. `view.OffRampConfig`